aloha-vw-jni/src/main/scala/com/eharmony/aloha/models/vw/jni/VwJniModel.scala

```diff
@@ -20,25 +21,24 @@ import com.eharmony.aloha.score.basic.ModelOutput
 import com.eharmony.aloha.score.conversions.ScoreConverter
 import com.eharmony.aloha.semantics.Semantics
 import com.eharmony.aloha.semantics.func.GenAggFunc
-import com.eharmony.aloha.util.{SimpleTypeSeq, EitherHelpers, Logging}
+import com.eharmony.aloha.util.{EitherHelpers, Logging, SimpleTypeSeq}
 import org.apache.commons.codec.binary.Base64
 import org.apache.commons.io.IOUtils
 import spray.json.{DeserializationException, JsValue, JsonReader, pimpAny, pimpString}
+import vw.learner._
 
 import scala.collection.immutable.ListMap
 import scala.collection.{immutable => sci, mutable => scm}
 import scala.util.{Failure, Success, Try}
 
-
 /**
  * Model that delegates to a VW JNI model.
  * @param modelId: ModelIdentity
- * @param vwJniModelSource either the FsInstance pointing to the VW model or the Base64 encoded VW model.
+ * @param modelSource either the FsInstance pointing to the VW model or the Base64 encoded VW model.
  * @param featureNames names of features (parallel to featureFunctions)
  * @param featureFunctions functions to extract values from the input value
  * @param defaultNs indices of features that will be placed in the default VW namespace
  * @param namespaces mapping from namespace name to indices of features that will be placed in the namespace
- * @param finalizer a function that transforms the native VW output type (Float) to B
```

Comment:
Sounds good.  Do you not like the interface?  I think it's pretty elegant given the restriction that `VWLearner` cannot be serialized.  That's worth a comment.